ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/QueryStreamWriter.java

```diff
@@ -70,16 +72,16 @@
 
     this.disconnectCheckInterval = disconnectCheckInterval;
     this.queryMetadata = ((QueuedQueryMetadata) queryMetadata);
-
+    this.queryMetadata.getOutputNode().setLimitHandler(new LimitHandler());
```

Comment:
could we put a similar method on `QueryMetadata` so we can avoid the chaining?

Reply:
Sure thing. Cost is 1 🍺 .

Reply:
lol